_data/team.yml

```diff
@@ -84,3 +84,14 @@
       - programming
       - video games
       - guitar
+      
+- name: Chloe Herman
+  title: PhD Student
+  pic: ChloeHerman.png
```

Comment:
Looks like this file hasn't been included yet in the pull request - can you make sure to `git add` and `git commit` that file, too?

Reply:
I think that I accidentally created two forks and committed it there. I think this fixed it. Let me know.